diff -Naur anaconda-21.48.22.93.orig/pyanaconda/ui/gui/hubs/progress.py anaconda-21.48.22.93/pyanaconda/ui/gui/hubs/progress.py
--- anaconda-21.48.22.93.orig/pyanaconda/ui/gui/hubs/progress.py	2016-09-05 07:10:25.000000000 -0500
+++ anaconda-21.48.22.93/pyanaconda/ui/gui/hubs/progress.py	2016-12-14 23:38:04.211360089 -0500
@@ -34,7 +34,7 @@
 from pyanaconda.flags import flags
 from pyanaconda import iutil
 from pyanaconda.constants import THREAD_INSTALL, THREAD_CONFIGURATION, DEFAULT_LANG, IPMI_FINISHED
-from pyanaconda.constants import eulaLocation
+#from pyanaconda.constants import eulaLocation
 from pykickstart.constants import KS_SHUTDOWN, KS_REBOOT
 
 from pyanaconda.ui.gui.hubs import Hub
@@ -137,7 +137,7 @@
 
         iutil.ipmi_report(IPMI_FINISHED)
 
-        self.set_warning(_("Use of this product is subject to the license agreement found at %s") % eulaLocation)
+        #self.set_warning(_("Use of this product is subject to the license agreement found at %s") % eulaLocation)
         self.window.show_all()
 
         # kickstart install, continue automatically if reboot or shutdown selected
