diff -Naur anaconda-21.48.22.134.orig/data/liveinst/gnome/fedora-welcome anaconda-21.48.22.134/data/liveinst/gnome/fedora-welcome
--- anaconda-21.48.22.134.orig/data/liveinst/gnome/fedora-welcome	2017-01-30 05:03:51.000000000 -0500
+++ anaconda-21.48.22.134/data/liveinst/gnome/fedora-welcome	2018-06-10 22:45:52.878406135 -0500
@@ -62,7 +62,7 @@
                                                 default_width: 600,
                                                 default_height: 550,
                                                 skip_taskbar_hint: true,
-                                                title: _("Welcome to Fedora"),
+                                                title: _("Bienvenido a DeskOS"),
                                                 window_position: Gtk.WindowPosition.CENTER });
       this.window.connect('key-press-event', Lang.bind(this,
           function(w, event) {
@@ -91,7 +91,7 @@
                                      spacing: 16 });
       tryContent.add(new Gtk.Image({ icon_name: 'media-optical',
                                      pixel_size: 256 }));
-      tryContent.add(makeLabel(_("Try Fedora"), true));
+      tryContent.add(makeLabel(_("Probar DeskOS"), true));
 
       let tryButton = new Gtk.Button({ child: tryContent });
       buttonBox.add(tryButton);
@@ -108,6 +108,7 @@
       buttonBox.add(installButton);
 
       this._label = makeLabel(_("You are currently running Fedora from live media.\nYou can install Fedora now, or choose \"Install to Hard Drive\" in the Activities Overview at any later time."), false);
+      this._label = makeLabel(_("Actualmente está ejecutando DeskOS desde un medio vivo.\nPuede instalar DeskOS ahora, o elegir \"Instalar en el disco duro\" posteriormente."), false);
       mainGrid.add(this._label);
 
       installButton.connect('clicked', Lang.bind(this,
